kubernetes/hdfs/values.yaml

```diff
@@ -46,6 +46,7 @@ namenode:
       resources:
         requests:
           storage: 10Gi
```

Comment:
Perhaps add a comment explaining the format of how to use this (variable.name: value)

Reply:
Added.

Reply:
Really sorry to be picky but maybe adding a comment above these examples saying: "Example usage:" might be good just to be really clear

Reply:
No problem at all, comment added.